src/lib/viewers/BaseViewer.js

```diff
@@ -579,6 +588,13 @@ class BaseViewer extends EventEmitter {
         if (this.fullscreenToggleEl && this.fullscreenToggleEl.focus) {
             this.fullscreenToggleEl.focus();
         }
+
+        if (this.focusTrap) {
+            this.focusTrap.destroy();
```

Comment:
Why do we need to destroy/recreate the focus trap each time we enter fullscreen mode?